src/Features/CSharpTest/InvertIf/InvertIfTests.Elseless.cs

```diff
@@ -483,6 +483,38 @@ void M()
             """);
     }
 
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/73917")]
+    public async Task IfWithoutElse_MoveSubsequentStatementsToIfBody4()
+    {
+        await TestAsync("""
+            public void SomeMethod()
+            {
+                object something = null;
+
+                [||]if (something == null)
+                {
+                    return;
+                }
+
+                #region A region
+                something = new object();
+                #endregion
+            }
+            """, """
+            public void SomeMethod()
+            {
+                object something = null;
+
+                if (something != null)
+                {
+                    #region A region
+                    something = new object();
+                    #endregion
+                }
+            }
+            """);
+    }
```

Comment:
can you have a test where the parent scope is a `case` block in a switch?  

also a test where we're inverting an if-statement as a top level statement?

Reply:
I added the switch case test 😃. 

I also checked for the top level if-statement but the refactor is not suggested. Am i missing something?